client/src/game/texture.js

```diff
@@ -14,12 +15,18 @@ class TextureService {
         padding: 3
       })
 
+      this.STAR_TEXTURE = new PIXI.Texture(PIXI.BaseTexture.from(require('../assets/stars/star.png')))
+
       // NEBULAS
       this.NEBULA_TEXTURES.push(new PIXI.Texture(PIXI.BaseTexture.from(require('../assets/nebula/neb1.png'))))
       this.NEBULA_TEXTURES.push(new PIXI.Texture(PIXI.BaseTexture.from(require('../assets/nebula/neb2.png'))))
       this.NEBULA_TEXTURES.push(new PIXI.Texture(PIXI.BaseTexture.from(require('../assets/nebula/neb3.png'))))
       this.NEBULA_TEXTURES.push(new PIXI.Texture(PIXI.BaseTexture.from(require('../assets/nebula/neb4.png'))))
```

Comment:
Remove the old nebula images if they are no longer used.

Reply:
they are used, if you disable draw stars then it reverts to using the old nebulas

Reply:
i didnt want to ditch them right away since some people might still prefer it, or they might just run better without all those individual stars, but so far i havent had any major performance issues on my laptop so we can probably ditch them latter...

Reply:
Alright, let's leave it as is 👍